tooling-e2e-tests/utils.sh

```
@@ -167,6 +167,21 @@ stop_relayer() {
+# ${2} - pocket money, ignored if ${1} is empty
+quote_relayer_fee() {
+  # if arg in not provided, default to native token
+  if [ $# -eq 0 ]; then
+    data='{"fee_token":"Native","pocket_money":"0"}'
+  else
+    data='{"fee_token":{"ERC20": "'${1}'"},"pocket_money":"'${2:-0}'"}'
+  fi
+  curl -s --header "Content-Type: application/json" \
+  --request POST \
+  --data "${data}" \
+  ${RELAYER_URL}/quote_fees | jq -r '.fee_details.total_cost_fee_token'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)